src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -200,8 +203,12 @@ private bool ValidOptions(ILogger logger)
 
         private TokenResult AccessToken(IPublicClientAuth publicClientAuth, CommandExecuteEventData eventData)
         {
+            var authParams = new AuthParameters(
+                AzureAuth.Ado.Constants.Client.VisualStudio,
+                this.Tenant,
+                new[] { AzureAuth.Ado.Constants.Scope.AzureDevOpsDefault });
             return publicClientAuth.Token(
-                AzureAuth.Ado.Constants.AdoParams,

```

Comment:
Does this mean we can get rid of this constant?

Reply:
Rather than get rid of it we might want to find a better way to use it. Perhaps a builder-style method on that allows overriding the tenant... 🤔

Reply:
I like that idea :)